api/node/Cargo.toml

```diff
@@ -19,13 +19,32 @@ build = "build.rs"
 [lib]
 crate-type = ["cdylib"]
 
+[features]
+default = ["backend-winit", "renderer-femtovg", "backend-qt", "accessibility"]
+# Keep in sync with features in nightly_snapshot.yaml, cpp_package.yaml, and slint_tool_binary.yaml
+# binaries: default = ["backend-winit", "renderer-femtovg", "renderer-skia", "accessibility"]
+# binaries-linux: default = ["backend-linuxkms-noseat", "backend-winit", "renderer-femtovg", "renderer-skia", "accessibility"]
```

Comment:
why noseat?

and is it not safe to enable this feature on non-linux?

Reply:
noseat because the Ubuntu version we're building against doesn't have libseat. We could upgrade to require a newer version of Ubuntu, but I'm not sure it's worth it at this point. Maybe when we mark the backend as not experimental anymore?

Good question about non-linux.... I think there were build issues, but you're right - maybe it can be fixed. I'll peek.